src/verso-manual/VersoManual/Docstring.lean

```diff
@@ -775,8 +775,8 @@ def docstring.descr : BlockDescr := withHighlighting {
 
     for altName in altNames do
       modify fun st => st
-        |>.saveDomainObject suggestionDomain name.toString id
-        |>.saveDomainObjectData suggestionDomain name.toString (json%{
+        |>.saveDomainObject suggestionDomain (altName.toString ++ "->" ++ name.toString) id
+        |>.saveDomainObjectData suggestionDomain (altName.toString ++ "->" ++ name.toString) (json%{
```

Comment:
This is a tiny detail, but perhaps this should be a Unicode arrow with spaces around it? It might make the JSON more readable for anyone implementing a tool.

Reply:
done!